userguide/dailyuse/libertine.rst

```diff
@@ -56,6 +56,14 @@ Remove a package::
 .. note::
     If you have more than one container, then you can use the option ``-i CONTAINER-IDENTIFIER`` to specify for which container you want to perform an operation.
 
+Run desktop application
+-----------------------
+
+To run a graphical application you installed whose package installed a desktop file, i.e. ``gedit``, run in a terminal::
```

Comment:
Maybe it's better to express that this is the way to start *from the command line*, which is not the normal way most users would do it. Also, maybe sentence could be split to be easier to read. How about 

To start a graphical application from the command line you can use ubuntu-app-launch. For example to start gedit, .....